project/ContinuousIntegration.scala

```diff
@@ -67,9 +67,7 @@ object ContinuousIntegration {
   /**
     * For "reasons" these projects are excluded from the root aggregation in build.sbt.
     */
-  private val unaggregatedProjects = Map(
-    "cwlEncoder" -> "not sure what this is"
-  )
+  private val unaggregatedProjects = Map.empty
```

Comment:
This change is not strictly related and more the result of auditing non-comment references to "cwl" in `.scala` files.